Patch-Source: https://github.com/pulumi/pulumi/commit/312929faeacba06d7655f471b5616a5aa7060a97
Backported for 3.170.0.
---
From 312929faeacba06d7655f471b5616a5aa7060a97 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <me@iwahbe.com>
Date: Wed, 17 Dec 2025 17:27:21 +0100
Subject: [PATCH] Do not assert that the pulumi/templates repo contains
 aws-javascript (#21280)

As of https://github.com/pulumi/templates/pull/1010,
github.com/pulumi/templates does not contain any template called
`aws-javascript`.
---
 pkg/cmd/pulumi/newcmd/new_test.go | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pkg/cmd/pulumi/newcmd/new_test.go b/pkg/cmd/pulumi/newcmd/new_test.go
index 4c5622327f9d..3dbe8cb6d16f 100644
--- a/pkg/cmd/pulumi/newcmd/new_test.go
+++ b/pkg/cmd/pulumi/newcmd/new_test.go
@@ -1021,7 +1021,6 @@ Available Templates:
   aws-fsharp                         A minimal AWS F# Pulumi program
   aws-go                             A minimal AWS Go Pulumi program
   aws-java                           A minimal AWS Java Pulumi program
-  aws-javascript                     A minimal AWS JavaScript Pulumi program
   aws-python                         A minimal AWS Python Pulumi program
   aws-scala                          A minimal AWS Scala Pulumi program
   aws-typescript                     A minimal AWS TypeScript Pulumi program
@@ -1128,7 +1126,6 @@ Available Templates:
   aws-fsharp                              A minimal AWS F# Pulumi program
   aws-go                                  A minimal AWS Go Pulumi program
   aws-java                                A minimal AWS Java Pulumi program
-  aws-javascript                          A minimal AWS JavaScript Pulumi program
   aws-python                              A minimal AWS Python Pulumi program
   aws-scala                               A minimal AWS Scala Pulumi program
   aws-typescript                          A minimal AWS TypeScript Pulumi program
